services/ext/api.go

```diff
@@ -394,10 +395,18 @@ func (api *PublicAPI) SetInstallationName(installationID string, name string) er
 }
 
 // Communities returns a list of communities that are stored
+// Deprecated: Use FetchSerializedCommunities instead
 func (api *PublicAPI) Communities(parent context.Context) ([]*communities.Community, error) {
 	return api.service.messenger.Communities()
 }
 
+// FetchSerializedCommunities returns a list of serialized communities.
+// The key difference from the Communities function is that it uses MediaServer
+// to construct image URLs for all the images rather than using base64 encoding.
+func (api *PublicAPI) FetchSerializedCommunities(parent context.Context) ([]json.RawMessage, error) {
```

Comment:
Just a naming comment, but usually, we use `fetch` when we need to get data from elsewhere (eg store nodes). I think calling this function just `SerializedCommunities` would be enough.